install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -84,6 +87,12 @@
   {{- $defaultK8sClientQPS = 10 -}}
   {{- $defaultK8sClientBurst = 20 -}}
 {{- end -}}
+
+{{- if or (.Values.gke.enabled) (eq .Values.routingMode "native") -}}
+  {{- $tunnelProto = "" -}}
+{{- else -}}
```

Comment:
The logic should be something like:

```
if (gke || routingMode == native) {
    tunnelProto = ""
} else if (values.tunnelProto != "" {
    tunnelProto = values.tunnelProto
} else {
    tunnelProto = vxlan
}
```

Make sense?

Reply:
Updated to match the above.